### ISSUE TITLE
[feature request] Configurable manual sort order

### ISSUE THREAD

**Issue by prokher** [pascalre/vscode-yaml-sort#8]
Dear @pascalre ,

I would like to ask if it is possible to add a configurable option with witch I could manually specify sort order for some keys. Let me explain. The main reason I use YAML sort plugin if to reduce difficulties during merging YAML-configuration across different git branches. It is much easier to merge when the order of keys is the same in the conflicting branches. Most of the YAML files I work with is either Kubernetes manifests or different their variations, e.g. Kustomize rules (`kustomization.yaml`). The thing is, in such files there is some natural order of keys which is not always alphabetical. I've seen #3 about Kubernetes-friendly sort, but I suppose it is better to have an option, so I can manually configure the plugin specifically for my project. Just as a motivating example, please, consider typical `kustomization.yaml`:

```yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
commonLabels: ...
patches: ...
resources: ...
```

For this type of files the "natural" sort order is the following:
```yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources: ...
patches: ...
commonLabels: ...
```
which obviously differs from the alphabetical order.

**Comment by pascalre** [comment]
Hi @prokher,
I like this issue. I will add an option to configure a list of keywords and add an extra command (e. g. "Custom sort YAML").

**Comment by pascalre** [comment]
Hi @prokher I am currently working on this feature.
Btw: I would really appreciate a rating on the marketplace for the extension. 👍

**Comment by pascalre** [comment]
@prokher 2.0.0 contains the feature!

**Comment by prokher** [comment]
Dear @pascalre when I try to setup a ... I get the following, I suppose it works fine, but message looks weird. What am I doing wrong?

<img width="393" alt="image" src="https://user-images.githubusercontent.com/399393/67243202-b78c3380-f45f-11e9-930d-5959cffc6539.png">

**Comment by pascalre** [comment]
Hi @prokher this was a configuration mistake. I fixed the wrong expected type in 02aa0030c455811c99993a5a9aea431a629544dc. Update 2.0.1 is available on the marketplace.
